src/views/Project/index.jsx

```diff
@@ -11,6 +11,7 @@ const data = [
     tag: 'JS',
     assignee: 'None',
     lastUpdated: '2018-05-20',
+    url: 'https://bugzilla.mozilla.org/show_bug.cgi?id=1436212',
```

Comment:
Note sure why we are adding these links when I believe this whole content is being removed in a separate PR.

Reply:
@eliperelman i think we need a link in the table entry so that user can click on the entry and get redirect to the bug page, be it in github or bugzilla.

Reply:
Having something hardcoded for now will be good for demoing the site during the all-hands.